api-tests/access-control/authed.test.js

```diff
@@ -43,7 +45,17 @@ multiAdapterRunners().map(({ before, after, adapterName }) =>
           }),
         {}
       );
-      items = await keystone.createItems(initialData);
+
+      for (const [listName, _items] of Object.entries(initialData)) {
+        const newItems = await createItems({
+          keystone,
+          listName,
+          items: _items.map(x => ({ data: x })),
```

Comment:
This sucks! `createItems` should completely abstract this `data: {}` BS - this confuses so many people.

Reply:
It sucks, but the right solution is to fix out GraphQL. Let's fix the root of the problem, rather than giving people two subtley different APIs to deal with.

Reply:
Cross posting my points from slack:

In this context it kinda is a different API already. Items is an array and why would people think this should be in the shape of `[{data:{ }]`

Yes, we should fix the queries, but create items is intended to hide the query syntax no?